app/src/main/java/com/example/campusguide/directions/ChooseDirectionOptionsDialogFragment.kt

```diff
@@ -92,6 +92,7 @@ class ChooseDirectionOptions constructor(private val route: Route?) : DialogFrag
                         CallbackDirectionsConfirmListener { start, end ->
                             //Display the directions time
                             route?.set(start, end)
+                            this.dismiss()
```

Comment:
🤦‍♂ so simple!

Reply:
Haha, yeah. Just needed to track down where it needed to go.